src/main/java/org/jvnet/hudson/plugins/thinbackup/ThinBackupPluginImpl.java

```diff
@@ -16,26 +16,23 @@
  */
 package org.jvnet.hudson.plugins.thinbackup;
 
-import antlr.ANTLRException;
-import hudson.Plugin;
-import hudson.scheduler.CronTab;
-import hudson.util.FormValidation;
+import hudson.Extension;
+import hudson.ExtensionList;
 import java.io.File;
 import java.util.logging.Level;
 import java.util.logging.Logger;
-import java.util.regex.Pattern;
-import java.util.regex.PatternSyntaxException;
+import jenkins.model.GlobalConfiguration;
 import jenkins.model.Jenkins;
+import net.sf.json.JSONObject;
 import org.apache.commons.lang.StringUtils;
+import org.jenkinsci.Symbol;
 import org.jvnet.hudson.plugins.thinbackup.utils.EnvironmentVariableNotDefinedException;
 import org.jvnet.hudson.plugins.thinbackup.utils.Utils;
-import org.kohsuke.stapler.QueryParameter;
 import org.kohsuke.stapler.StaplerRequest;
-import org.kohsuke.stapler.StaplerResponse;
 
-public class ThinBackupPluginImpl extends Plugin {
-
-    private static final int VERY_HIGH_TIMEOUT = 12 * 60;
+@Extension
+@Symbol("ThinBackup")
```

Comment:
I think we typically use lower camel case for symbols in Jenkins symbols. Is there a particular reason we want this to be in upper camel case?

```suggestion
@Symbol("thinBackup")
```